tokens_definition_example_draft_v2.json

```diff
@@ -0,0 +1,205 @@
+{
+	"tokenCategories" : [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "layout",
+			"name": "layout"
+		}
+	],
+	"tokenSets" : [
+		{
+			"label": "grid",
+			"name": "grid"
+		},
+		{
+			"label": "links",
+			"name": "links"
+		}
+	],
+	"tokens": [
+		{
+			"mappings" : [
+				{
+					"name": "CSSVariable",
+					"outputKey": "$link-color",
+					"outputValues": [
+						{
+							"tokenValue": "Red",
+							"mappedValue": "#FF0000"
+						},
+						{
+							"tokenValue": "Salmon",
+							"mappedValue": "#FA8072"
+						}
+					]
+				}
+			],
+			"editorType": "ColorPicker",
+			"label": "link-color",
+			"name": "linkColor",
+			"validValues": [
+				{
+					"label": "salmon",
+					"value" : "Salmon"
+				},
+				{
+					"label": "red",
+					"value" : "Red"
+				}
+			],
+			"tokenCategoryName":  "general",
+			"tokenSetName": "links",
```

Comment:
Similarly, if we dropped the `token` namespace maybe this could just be `category` and `set`... I would also prefer to drop the `Name` suffix... based on this, I don't think this can be easily confused with anything else. It seems obvious we're using `name` as the key for `sets` and `categories`, so wouldn't `category: "general"` make sense and be less verbose?

Another thing is if we want to use `name` as the key for those or something more explicit lik `id`

Reply:
I favor explicitly stating which attribute is being used for the reference (`categoryName`, `categoryKey`, `categoryId`... rather than a implicit `category`)

Regarding what name to use as key maybe `key` is clearer than `name` and would make it more clear that it needs to be unique.

Reply:
Sounds good to me